docs/sdk/v3/guides/liquidity/02-modifying-position.md

```diff
@@ -0,0 +1,90 @@
+---
+id: modifying-liquidity
+title: Adding & Removing Liquidity
+---
+
+## Introduction
+
+This guide will cover how to modify a liquidity position by adding or removing liquidity on the Uniswap V3 protocol.
+It is based on the [modifying a position code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/modifying-position), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+To run this example, check out the examples's [README](https://github.com/Uniswap/examples/blob/main/v3-sdk/modifying-position/README.md) and follow the setup instructions.
+
+:::info
+
+If you need a briefer on the SDK and to learn more about how these guides connect to the examples repository, please visit our [background](./01-background.md) page!
+
+:::
+
+In the Uniswap V3 protocol, liquidity positions are represented using non-fungible tokens. In this guide we will use the `NonfungiblePositionManager` class to help us mint a liquidity position and then modify the provided liquidity for the  **USDC - DAI** pair. The inputs to our guide are the **two tokens** that we are pooling for, the **amount** of each token we are pooling for, the Pool **fee** and the **percentage** by which to **add and remove** from our position.
+
+The guide will **cover**:
+1. Adding liquidity to our position
+2. Removing liquidity from our position
+
+Please note tha the step of creating a position is covered in great detail in the previous guide, which focuses on [minting a position](./01-minting-position.md).
+
+At the end of the guide, given the inputs above, we should be able to mint a liquidity position with the press of a button and view the position's id on the UI of the web application. We should also be able to add or remove liquidity with the press of a button and see the change reflected in the balance of our tokens.
+
+## Example
+
+### Adding liquidity to our position by using the `NonfungiblePositionManager`'s `addCallParameters` to get the data for making the add liquidity transaction, and then executing the transaction
```

Comment:
Missed this on the last one, we should shorten these titles to be simpler / single line / readable!